libcrux-ml-kem/src/vector/portable/ntt.rs

```diff
@@ -1,6 +1,18 @@
 use super::arithmetic::*;
 use super::vector_type::*;
 
+#[inline(always)]
+pub(crate) fn ntt_step(
+    v: &mut PortableVector,
+    zeta: i16,
+    i: usize,
+    j:usize
+) {
+    let t = montgomery_multiply_fe_by_fer(v.elements[j], zeta);
+    v.elements[j] = v.elements[i] - t;
+    v.elements[i] = v.elements[i] + t;
+}
```

Comment:
`i` and `j` could be `const` here. Not sure if it makes a difference though.

Reply:
Perf does not seem to be affected, probably because this is `inline(always)`.